lib/xcodeproj/project/object_attributes.rb

```diff
@@ -316,6 +316,14 @@ def attribute(name, klass, default_value = nil)
             define_method("#{attrb.name}=") do |value|
               @simple_attributes_hash ||= {}
               attrb.validate_value(value)
+
+              existing = @simple_attributes_hash[attrb.plist_name]
+              if existing.is_a?(Hash) && value.is_a?(Hash)
```

Comment:
no, they can't be returns since you're inside a Proc here, and you'd get something like a NonLocalJumpError